tools/asserts.go

```diff
@@ -0,0 +1,133 @@
+package tools
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	mcpgrafana "github.com/grafana/mcp-grafana"
+	"github.com/mark3labs/mcp-go/server"
+)
+
+func newAssertsClient(ctx context.Context) (*Client, error) {
+	grafanaURL, apiKey := mcpgrafana.GrafanaURLFromContext(ctx), mcpgrafana.GrafanaAPIKeyFromContext(ctx)
+	url := fmt.Sprintf("%s/api/plugins/grafana-asserts-app/resources/asserts/api-server", strings.TrimRight(grafanaURL, "/"))
+
+	client := &http.Client{
+		Transport: &authRoundTripper{
+			apiKey:     apiKey,
+			underlying: http.DefaultTransport,
+		},
+	}
+
+	return &Client{
+		httpClient: client,
+		baseURL:    url,
+	}, nil
+}
+
+type GetAssertionsParams struct {
+	StartTime  int64  `json:"startTime" jsonschema:"description=The start time of the assertion status in unix milliseconds format"`
+	EndTime    int64  `json:"endTime" jsonschema:"description=The end time of the assertion status in unix milliseconds format"`
```

Comment:
FWIW elsewhere in the server we use `time.Time` and mention that the timestamp should be in RFC3339 in the description, which tends to work quite well (it's easier for a model to get an RFC3339 timestamp correct than a unix timestamp, I guess)

Reply:
Oh. If that's the case, I can revert back to use RFC3339 and do the `*1000` when calling Asserts API.